qt/widgets/common/src/AlgorithmPropertiesWidget.cpp

```diff
@@ -279,6 +279,12 @@ bool isCalledInputWorkspace(PropertyWidget *const candidate) {
   return propertyName == "InputWorkspace";
 }
 
+bool isCalledLHSWorkspace(PropertyWidget *const candidate) {
+  Mantid::Kernel::Property const *const property = candidate->getProperty();
+  const std::string &propertyName = property->name();
+  return propertyName == "LHSWorkspace";
```

Comment:
I think it might be possible to have a slight speed-up here if we put this line in the function above e.g. 
```suggestion
  return propertyName == "InputWorkspace"   || propertyName == "LHSWorkspace";
```

as we wouldn't need to call `getProperty` and `name` twice whenever we open the algorithm dialog. We could then just call one of the functions, and change the function name